swagger_test_class.js

```diff
@@ -8,6 +8,9 @@ function _post(request, response) {
       .send(response.exampleRequest)
       .end(function (err, res) {
         expect(res.statusCode).to.be.equal(response.status);
+        if (res.body.modified){
```

Comment:
Rather than removing specific fields in the code, create a more generic function that can accept a list of fields to remove, and then run that function against the response.